packages/components/src/date-time/date/styles.ts

```diff
@@ -17,12 +17,24 @@ export const Wrapper = styled.div`
 `;
 
 export const Navigator = styled( HStack )`
+	display: grid;
+	grid-template-columns: 0.5fr repeat( 5, 1fr ) 0.5fr;
+	justify-items: center;
 	margin-bottom: ${ space( 4 ) };
 `;
 
+export const ViewPreviousMonthButton = styled( Button )`
+	grid-column: 1 / 2;
+`;
+
+export const ViewNextMonthButton = styled( Button )`
+	grid-column: 7 / 8;
+`;
```

Comment:
Minor: I wondered if this could be a single, props-parameterized component like `NavigatorButton` with props `isNext` / `isPrevious`, but maybe this is clearer as it is.